lib/compiler.js

```diff
@@ -640,6 +662,21 @@ function buildAllMixinInstructions(appRootDir, mixinDirs, options) {
   return mixins;
 }
 
+function fetchMixinsUsedInModels(modelInstructions) {
```

Comment:
Please use different names for "a list of mixin instructions" and "a list of mixin names". Since there are no types in javascript, we must be more careful about method naming to make their purpose clear.

In this case, I am proposing `fetchMixinNamesUsedInModelInstructions`.